docs/travis.md

```diff
@@ -1,25 +0,0 @@
-# How we use Travis CI
```

Comment:
Do you want to delete this while you still have a `.travis.yml` around?

Reply:
Yes, we probably do.